Modules/EMCAL/src/ClusterTask.cxx

```diff
@@ -60,13 +67,32 @@ ClusterTask::~ClusterTask()
 
 void ClusterTask::initialize(o2::framework::InitContext& /*ctx*/)
 {
+  QcInfoLogger::setDetector("EMC");                        //svk
   ILOG(Info, Support) << "initialize ClusterTask" << ENDM; // QcInfoLogger is used. FairMQ logs will go to there as well.
 
   // this is how to get access to custom parameters defined in the config file at qc.tasks.<task_name>.taskParameters
   if (auto param = mCustomParameters.find("myOwnKey"); param != mCustomParameters.end()) {
     ILOG(Info, Devel) << "Custom parameter - myOwnKey: " << param->second << ENDM;
   }
 
+  auto get_bool = [](const std::string_view input) -> bool { //svk
+    return input == "true";
+  };
+
+  if (hasConfigValue("useInternalClusterizer")) {
+    mInternalClusterizer = get_bool(getConfigValueLower("useInternalClusterizer"));
+    if (mInternalClusterizer) {
+      LOG(debug) << "Enabling internal clusterizer . . . " << ENDM;
```

Comment:
Change to ILOG(Info, Support) -> Settings can be shown in the infoBrowser

Reply:
Changed to ILOG(Info, Support).